toolchain/install/busybox_info.h

```diff
@@ -24,21 +27,66 @@ struct BusyboxInfo {
 // Returns the busybox information, given argv[0]. This primarily handles
 // resolving symlinks that point at the busybox.
```

Comment:
Sure, updated comment a bit.

Reply:
For what it's worth, I don't think the new comment is specific enough to have helped me understand the implementation or the tests (e.g. I don't think I'd know what "most consistent" means, and I particularly wouldn't expect it to mean that shallower symlink traversals are preferred). However, that level of specificity may not be what other readers need, especially when the function is doing something heuristic like this.

Reply:
I've tried a new version of the comment. Is it any better?

Reply:
Just to be more explicit: I don't think this is blocking, because I'm not sure how much my experience as a reader generalizes, and I'm not sure how much effort it's worth putting into wordsmithing this API comment.

The previous revision worked better for me than this one, because it focused on the contract (albeit in imprecise terms), whereas this focuses on the implementation, and the implementation is complex enough that it's not clear what it implies about the contract.

If you want to keep iterating on this comment, maybe something like this?

"If `argv0` is the path of a busyboxed executable that is part of a valid Carbon install, or indirectly refers to it via symlinks, then `bin_path` will be the path of the `lib/carbon/carbon-busybox` file in the same Carbon install."

I'm not sure that fully captures the intent, but that's the kind of contract-focused specificity that would have helped me.

Reply:
Sure, tried wording much closer to that.

And it wasn't clear any of the comments were non-blocking, mostly trying to make progress here as this PR has been in review for a long time now.

Reply:
> Sure, tried wording much closer to that.

Looks good, thanks.

> And it wasn't clear any of the comments were non-blocking, mostly trying to make progress here as this PR has been in review for a long time now.

Sorry, I'll try to be more explicit about that.